features/plugin-activate.feature

```diff
@@ -106,3 +106,12 @@ Feature: Activate WordPress plugins
       """
       1 out of 1
       """
+
+  Scenario: Adding --exclude with plugin activate --all should exclude the plugins specified via --exclude
+    When I try `wp plugin activate --all --exclude=hello`
+    Then STDOUT should be:
+      """
+      Plugin 'akismet' activated.
+      Success: Activated 1 of 1 plugins.
+      """
+    And the return code should be 0
```

Comment:
Nit: Missing newline at the end of the file

Reply:
d03e01b7